deploy/openelb.yaml

```diff
@@ -1122,6 +1122,14 @@ subjects:
   namespace: openelb-system
 ---
 apiVersion: v1
+data:
+  key: b3BlbmVsYi1zcGVha2Vycw==
```

Comment:
Just a question, where did the `b3BlbmVsYi1zcGVha2Vycw==` of this key come from?

Reply:
It is the base64 encoding of "openelb-speakers". As you can see, I use "openelb-speakers" as the default secret for the memberlist in OpenELB. Of course, you can also customize the secret by modifying the Secret resource.